site/docs/hive.md

```diff
@@ -79,12 +79,45 @@ In order to query a Hive table created by either of the HiveCatalog methods desc
 ```sql
 SET iceberg.mr.catalog=hive;
 ```
-You should now be able to issue Hive SQL `SELECT` queries using the above table and see the results returned from the underlying Iceberg table. Both the Map Reduce and Tez query execution engines are supported.
```

Comment:
Why did we remove that both MR and Tez is supported?

Reply:
> Why did we remove that both MR and Tez is supported?

I moved it to the bottom under "Features" so I didn't have to keep repeating it :)

Reply:
It has been moved down here: https://github.com/apache/iceberg/pull/1837/files#diff-0270f04c6c1a4be5da895415fff2797103da7ded6ec97c303f2f7e218e99ac26R122 to avoid duplication :)

Reply:
Ok guys, just do not pile on me please 😄
I am reading stuff sequentially 😄